adm_my_files/config_example.php

```diff
@@ -47,3 +47,7 @@
 // for an update of the Admidio database to a new version of Admidio.
 // For a more comfortable and easy update you can set this preference = 0.
 $gLoginForUpdate = 1;
+
+// Set the preferred password hashing algorithm. Default is the PHP default "PASSWORD_DEFAULT"
+// Possible values are: PASSWORD_DEFAULT, PASSWORD_BCRYPT, 'SHA512'
```

Comment:
because there is no PHP default constant for SHA512 as int. So i think self defined values should be strings to get no conflict in future.
and you say it: we haven't include our constants here